dev/NavigationView/TestUI/NavigationViewRS4Page.xaml

```diff
@@ -36,6 +36,9 @@
                 PaneOpened="NavView_PaneOpened"
                 BackRequested="NavView_BackRequested"
                 AccessKey="H">
+            <!-- Note: Adding more NavViewItems here will cause the following tests to fail:
```

Comment:
>fail [](start = 87, length = 4)

I'd prefer we explain why here as well so we don't think this is a bug worth investigating.

Reply:
Done.